src/test/java/com/spotify/docker/AbstractDockerMojoTest.java

```diff
@@ -93,10 +96,13 @@ public void setUp() {
   @Test
   public void testDockerHostSet() throws Exception {
     ReflectionTestUtils.setField(sut, "dockerHost", DOCKER_HOST);
+    ReflectionTestUtils.setField(sut, "dockerCertPath", DOCKER_CERT_PATH);
```

Comment:
it would also be fine to make the field `protected` and add `@VisibleForTesting` to it